docs/docs/technology/introduction.md

```diff
@@ -87,15 +87,22 @@ flowchart LR
 
 #### **Main Translation Steps:**
 
-1. **text-to-SignWriting Translation**: Spoken language text is translated into SignWriting sequences using machine translation (it sees a lot of data, and extrapolates from it).
+1. **text-to-SignWriting Translation**: Spoken language text is translated into SignWriting sequences using machine translation (it sees a lot of data, and extrapolates from it). Timothy Comment: What is the conntext for this? are you explaining something to the reader here? 
```

Comment:
do not leave comments in the body of the documentation, leave comments, like i do, using github's comment system

Reply:
and to answer your question: yes, this line explains the first part of the pipeline